create_salsa_project.go

```diff
@@ -0,0 +1,166 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+)
+
+const (
+	salsaGroupID = 2638
+	salsaApiUrl  = "https://salsa.debian.org/api/v4"
+)
+
+type gitlabProject struct {
+	ID            int    `json:"id"`
+	Name          string `json:"name"`
+	HttpUrlToRepo string `json:"http_url_to_repo"`
+}
+
+func execCreateSalsaProject(args []string) {
+	projectName := getProjectNameOrExit(args)
+	token := getSalsaTokenOrExit()
+
+	gitlabProject, err := createSalsaProject(projectName, token)
+
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Could not create %s on salsa: %s\n", projectName, err)
```

Comment:
Here and elsewhere, instead of fmt.Fprintf+os.Exit, use log.Fatalf.

Reply:
Done.